CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 * Upgraded to Realm Sync 1.5.2.
 
+### Bug Fixes
```

Comment:
It seems the changelog is from the latest. But why didn't github show a conflict warning?